js/app.js

```diff
@@ -1,3 +1,71 @@
 'use strict';
+/**
+ * Construct and fill Product Array
+ */
+var urls = ['../img/bag.jpg', '../img/banana.jpg', '../img/bathroom.jpg', '../img/boots.jpg', '../img/breakfast.jpg', '../img/bubblegum.jpg', '../img/chair.jpg', '../img/cthulhu.jpg', '../img/dog-duck.jpg', '../img/dragon.jpg', '../img/pen.jpg', '../img/pet-sweep.jpg', '../img/scissors.jpg', '../img/shark.jpg', '../img/sweep.png', '../img/tauntaun.jpg', '../img/unicorn.jpg', '../img/usb.gif', '../img/water-can.jpg', '../img/wine-glass.jpg'];
```

Comment:
oooooooooh this is why your images aren't working! Like I mentioned yesterday, this is a difference between using URLs in CSS and JS: in CSS, you use the path of the CSS file as the root, so you need a `..`. In JS, that file is pulled in and it's *as if* it were right there between the script tags in `index.html`. Because of that, you don't need the `..`s: you just want `img/name.jpg` for all of these.